cmd/monaco/cmdutils/cmdutils.go

```diff
@@ -39,12 +39,14 @@ var ssoTokenURL = "https://sso.dynatrace.com/sso/oauth2/token" //nolint:gosec
 // Depending on the configured environment "type" the function tries to call the version endpoint of either
 // classic gen or platform gen. The function will return an error as soon as
 // it receives an error from calling the version endpoint of an environment
-func VerifyClusterGen(envs manifest.Environments) error {
+func VerifyClusterGen(envs manifest.Environments) bool {
 	for _, env := range envs {
 		switch env.Type {
 		case manifest.Classic:
 			if _, err := client.GetDynatraceVersion(client.NewTokenAuthClient(env.Auth.Token.Value), client.Environment{URL: env.Url.Value, Type: client.Classic}); err != nil {
-				return fmt.Errorf("could not verify Dynatrace cluster generation of environment %q (%q). Please check the configured Auth credentials in the manifest", env.Name, env.Url)
+				log.Error("Could not authorize against the environment with name %q (%s) using token authorization. "+
+					"Please verify that this environment is a classic generation cluster.", env.Name, env.Url.Value)
```

Comment:
```suggestion
					"Please verify that this environment is a classic Dynatrace environment.", env.Name, env.Url.Value)
```

We're not interacting with the cluster, but an environment/tenant - saying environment twice sounds a bit strange, but I couldn't come up with any better suggestion